inc/conversion.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef INC_CONVERSION_HPP_
+#define INC_CONVERSION_HPP_
+#include <iostream>
+#include <fstream>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <string>
+using namespace std;
+
+class Convert
+{
+	private:
+		string incoming;
+		int byts;
+	protected:
+	public:
+		Convert();
```

Comment:
open the file in the constructor you will be given a string name from outside, open to the file in std::ios_base::app mode